homeassistant/components/airvisual/strings.json

```diff
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "title": "AirVisual",
+    "step": {
+      "user": {
+        "title": "Configure AirVisual",
+        "description": "Monitor air quality in a geographical location.",
+        "data": {
+          "api_key": "API Key",
+          "latitude": "Latitude",
+          "longitude": "Longitude",
+          "show_on_map": "Show monitored geography on the map"
+        }
+      }
+    },
+    "error": {
+      "identifier_exists": "API key already registered",
+      "invalid_api_key": "Invalid API key"
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "description": "Configure options for the AirVisual integration.",
```

Comment:
Bram added support for options titles just a couple of days back

Reply:
Roger – I'll remember this when I tackle options in a future PR.